src/translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
+  "socialProtection.benefitPlan.delete.confirm.title": "DELETE BENEFIT PLAN {code} {name}",
+  "socialProtection.benefitPlan.delete.confirm.message": "Are you sure you want to delete the benefit plan?",
+  "socialProtection.benefitPlan.delete.mutationLabel": "Delete benefit plan {name}",
+  "socialProtection.benefitPlan.update.mutationLabel": "Update product {name}",
+  "socialProtection.benefitPlan.editButtonTooltip": "Edit",
+  "socialProtection.benefitPlan.deleteButtonTooltip": "Delete",
+  "socialProtection.benefitPlan.searcherResultsTitle": "{benefitPlansTotalCount} Benefit Plans",
+  "socialProtection.benefitPlan.dateValidTo": "Date to",
+  "socialProtection.benefitPlan.dateValidFrom": "Date from",
+  "socialProtection.benefitPlan.name": "Name",
+  "socialProtection.benefitPlan.code": "Code",
+  "socialProtection.benefitPlan.maxBeneficiaries": "Max Beneficiaries",
+  "socialProtection.benefitPlan.json_ext": "JSON_EXT",
```

Comment:
Maybe it could be good to stick to the convention - pascal case. Please transform 'json_ext' -> jsonExt

Reply:
doone